### DISCUSSION TITLE
New printer profiles

### DISCUSSION THREAD

**Discussion by bttguy** [sn4k3/UVtools#134] [Ideas]
Have you considered adding a "no support" version for the profiles? It would handy for models that require no support neither a pad. Its just a few clicks to create and save them for myselft, but it might be useful in the long run for others as well.

And my second thought is would it be possible to create some universal baseline profiles for resins, that corresponds to the printer added (RGB vs monoschrome at least). Its just a possible pitfall, that I thankfully realised before exported, that the Prusa resin profile is for RGB with insane exposure times for my monochrome LD002H. It would be a nice touch try to adress this. I am very new to resin printing, and it would be a nice starting point for us, beginners.

Just try to help from a beginners perspective how this tool could be even better. But seriously, this is a game changer. Did my first print in Chitu, just for the sake of a "factory stock print", but since then never looked back thanks to you! Keep it up!

**Comment by sn4k3** [comment]
> Have you considered adding a "no support" version for the profiles? It would handy for models that require no support neither a pad. Its just a few clicks to create and save them for myselft, but it might be useful in the long run for others as well.


You mean on PrusaSlicer? If is the case there are no need to introduce noise on profile box, that can be easly done by select **None** on select box near printer profile. It's easier than have a ton of new profiles just to remove the supports...

![image](https://user-images.githubusercontent.com/113281/105618196-90e44780-5ddc-11eb-888f-240c3c0c61b1.png)


> And my second thought is would it be possible to create some universal baseline profiles for resins, that corresponds to the printer added (RGB vs monoschrome at least). Its just a possible pitfall, that I thankfully realised before exported, that the Prusa resin profile is for RGB with insane exposure times for my monochrome LD002H. It would be a nice touch try to adress this. I am very new to resin printing, and it would be a nice starting point for us, beginners.

Resin profiles must be added per printer/resin/layer height combo, there are too many variables for me to build them myself without any printer and resins to test with, this is up to users, they must build their tablet no matter what slicer they use, even chitubox stock settings may not be perfect, that require user testing, same resin brand can have diferent times. But well i can add a generic with averages times so users can clone and tune, but i always advice to test and tune.
Something i can also do is allow users to send me their tuned profiles and i add them to the correct printer/resin.
